drafts/inlinemacros.md

```diff
@@ -165,99 +170,66 @@ positions:
 Inline types can specifically not be used as
 
  - types of abstract methods or fields
- - types of variables
- - types of class parameters
+ - types of vars
+ - types of parameters to non-inline classes
  - type parameters to methods
  - type parameters in `new` expressions or class parents.
 
-## Inline Overloading Resolution
```

Comment:
Why would it?
Lets say you have on `List`:
``
def sum(implicit Numeric[T]): Int
inline def sum(this: List[Int]): Int
```
sum is an overloaded method in this case.